src/stories/Foundation/PanTilt/index.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import { BlueTonesEnum } from '../../../shared/colors';
+import styled from 'styled-components';
+
+const Wrapper = styled.div<{ width: number; height: number }>`
+  background: var(--color-grey91);
+  width: ${(p) => `${p.width}px`};
+  height: ${(p) => `${p.height}px`};
+  border-radius: var(--spacing-8);
+`;
+
+const StyledRect = styled.rect`
+  cursor: move;
+`;
+
+const StyledPath = styled.path`
+  cursor: move;
+`;
+
+const getArrowTransformation = (x, y, scale) =>
+  `translate(${x},${y}) scale(${scale})`;
```

Comment:
Maybe consider having a threshold for how low the scale value can be. Arrows might be on the slim side when they get scaled down alot.